hackathon/api/views.py

```diff
@@ -70,26 +72,42 @@ def get_queryset(self):
         return HackathonEvent.objects.all()
 
 
-class ProblemCategoryClass(HttpResponse):
+
+class ProblemCategoryClass(generics.ListAPIView):
+    permission_classes = []
     serializer_class = ProblemCategorySerializer
 
-    def get_problem_category(request):
-        if request.method == 'GET':
-            category = ProblemCategory.objects.all().values('id','category','description','created_at')
-            # serializer = ProblemCategorySerializer(category,many=True)
-            listed = list(category)
-            return JsonResponse(listed, safe=False)
-        else:
-            print("Only GET is allowed")
-
-    def get_problem_statements(request, category_id):
-        if request.method == 'GET':
-            statements = ProblemStatement.objects.filter(category=category_id).values("id","statement","description","created_at","category_id")
-            # serializer = ProblemCategorySerializer(category, many=True)
-            listed = list(statements)
-            return JsonResponse(listed, safe=False)
-        else:
-            print("Only GET is allowed")
+    def get_queryset(self):
+        return ProblemCategory.objects.all()
+
+
+class ProblemStatementClass(generics.ListAPIView):
+    permission_classes = []
+    serializer_class = ProblemStatementSerializer
+
+    def get_queryset(self,):
+        category_id = self.kwargs['category_id']
+        print(category_id)
+        return ProblemStatement.objects.filter(category=category_id)
+
+
+
+import json
+
+def Send_to_Email(request):
+    email_to = "harshzf2@gmail.com"
+    subject = 'Thank you for registering to our site'
+    message = ' message'
+    email_from = "taacropolis@gmail.com"
```

Comment:
Use this in environment variable like:

1. In settings define `EMAIL_FROM_SETTING = config('EMAIL_FROM_ID')`
2. Import and use it in settings.py file

And in your case you have "EMAIL_HOST_USER"
I know I said static but personal emails and password needs to be in env